addons/website_mass_mailing/migrations/12.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,7 @@
+---Fields in module 'website_mass_mailing'---
+---XML records in module 'website_mass_mailing'---
+DEL ir.ui.view: website_mass_mailing.page_unsubscribe
+DEL ir.ui.view: website_mass_mailing.page_unsubscribed
+DEL ir.ui.view: website_mass_mailing.unsubscribe
+DEL ir.ui.view: website_mass_mailing.unsubscribed
+# DONE: mass_mailing post-migration: renamed to mass_mailing
```

Comment:
This is switched: this text must be in `mass_mailing`

Reply:
Yes, sorry, the names are so similar that I got confused... :dizzy_face: